src/commands/init.ts

```diff
@@ -18,10 +18,16 @@ import {
   defaultDir,
   defaultManifestFileName,
```

Comment:
This file looks very complex, woul be a great improvement to split it so its easier to review and edit in the future. Even considering creating modules for each command if required

Reply:
I have created a module to reorganize `init` command and make it simpler